### PR TITLE
Bug 1089920 - use fixed gaia header

### PR DESCRIPTION
**Pull request by julienw**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[julienw Julien Wajsberg (julienw)](https://github.com/julienw) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=d2ac7ca36754)

**Comment by try-server-hook** [comment]
[julienw Julien Wajsberg (julienw)](https://github.com/julienw) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=ff21a80a7bed)

**Comment by try-server-hook** [comment]
[julienw Julien Wajsberg (julienw)](https://github.com/julienw) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=ca93b2126a23)

File: apps/sms/js/startup.js [hunk 1]
```diff
@@ -74,14 +74,24 @@ var Startup = {
       window.performance.mark('objectsInitEnd');
       PerformanceTestingHelper.dispatch('objects-init-finished');
     });
+    this._initHeaders();
+    return lazyLoadPromise;
+  },
+
+  _initHeaders: function() {
+    console.log('init headers');
```
Comment (steveck-chung) [inline apps/sms/js/startup.js]: nit: remove log

File: apps/sms/index.html [hunk 2]
```diff
@@ -120,8 +120,8 @@
   <body role="application" class="skin-comms theme-communications">
     <article id="main-wrapper" class="wrapper" data-position='right'>
       <section id="thread-list" class="panel" role="region">
-        <gaia-header class="view-header regular-header">
-          <h1 data-l10n-id="messages"></h1>
+        <gaia-header class="view-header regular-header" title-start="0" title-end="100">
```
Comment (steveck-chung) [inline apps/sms/index.html]: nit: maybe some comment about this title-start/title-end?

File: apps/sms/index.html [hunk 3]
```diff
@@ -120,8 +120,8 @@
...
+        <h1 data-l10n-id="messages"></h1>
```
Comment (steveck-chung) [inline apps/sms/index.html]: Indention?

**Comment by try-server-hook** [comment]
[julienw Julien Wajsberg (julienw)](https://github.com/julienw) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=2e7b69338c0c)

**Pull request merged**